test_rclcpp/CMakeLists.txt

```diff
@@ -135,6 +137,46 @@ if(AMENT_ENABLE_TESTING)
         "rclcpp")
     endif()
 
+    if(TARGET local_parameters_cpp__${middleware_impl})
+      ament_add_gtest(local_parameters_cpp__${middleware_impl} "test/local_parameters.cpp")
```

Comment:
There's an argument to SKIP_LINKING_MAIN_LIBRARIES I'll take care of that and clean up the whitespace above before we merge.